File: ios/FlankerSandboxView/FlankerSandboxViewManager.m
//
//  FlankerSandboxViewManager.m
//  MindloggerMobile
//
//  Created by Dzmitry on 16.03.23.
//

#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(FlankerSandboxViewManager, RCTViewManager)
  RCT_EXPORT_VIEW_PROPERTY(onClick, RCTBubblingEventBlock)
  RCT_EXPORT_VIEW_PROPERTY(imageUrl, NSString)
  RCT_EXPORT_VIEW_PROPERTY(message, NSString)
@end

File: ios/FlankerSandboxView/FlankerSandboxViewManager.swift
//
//  FlankerSandboxViewManager.swift
//  MindloggerMobile
//
//  Created by Dzmitry on 16.03.23.
//

@objc(FlankerSandboxViewManager)
class FlankerSandboxViewManager: RCTViewManager {
  override static func requiresMainQueueSetup() -> Bool {
     return true
   }
  
   override func view() -> UIView! {
     return FlankerSandboxView()
   }
}
